bootstrap/kubeadm/config/crd/bases/bootstrap.cluster.x-k8s.io_kubeadmconfigs.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
```

Comment:
We should audit our APIs for multi-line TODOs and make them single line, see: https://github.com/kubernetes-sigs/controller-tools/pull/877 (PR description)

Atm we still end up with half the TODO in the CRD, which is not great :)